tutorials/asr/Streaming_ASR.ipynb

```diff
@@ -309,12 +309,10 @@
                 "import soundfile as sf\n",
                 "def get_samples(audio_file, target_sr=16000):\n",
                 "    with sf.SoundFile(audio_file, 'r') as f:\n",
-                "        dtype = 'int16'\n",
                 "        sample_rate = f.samplerate\n",
-                "        samples = f.read(dtype=dtype)\n",
+                "        samples = f.read()\n",
                 "        if sample_rate != target_sr:\n",
-                "            samples = librosa.core.resample(samples, sample_rate, target_sr)\n",
-                "        samples=samples.astype('float32')/32768\n",
+                "            samples = librosa.core.resample(samples, orig_sr=sample_rate, target_sr=target_sr)\n",
```

Comment:
We have one more of get_sample in this tutorials:
buffered_transducer_inference